docs/source/DeveloperDocumentation/TaskEngineWorkflowFeatures.md

```diff
@@ -391,3 +391,71 @@ The AVOD example below shows three clips being stitched together. The first clip
   }
 }
 ```
+
+```json
+
+```
+
+## LIVE COMPOSE WITH MANIFEST MANIPULATION
+
+The Vualto Task Engine now supports manifest manipulation to generate LIVE COMPOSE streams (VOD playlist looping), using AWS Mediatailor Channel Assembly. This workflow takes VOD streaming URLs directly and the resulting Live stream fragments come frome the original VOD streaming URLs - this could result in cost savings in caching layers in some instances.
+
+With this workflow, it is also possible to condition live streams for SSAI (server side ad insertion), however, it requires ad slate clips (also in the form of VODs) in order to signal ad breaks. The slate clips are stitched linearly with the clip sources at the given `presentation_time` (relative to the clip source) and the relevant ad break timed metadata added to the resulting live stream. If no `presentation_time` is provided, it will default to `00:00:00` (pre-roll).
```

Comment:
Here you use "live" whereas above you use "Live", no particular preference from me but it should be consistent (ideally across all our documentation and this is probably where writing guidelines would come in).